### ISSUE TITLE
Clarify NEO smart contract java development

### ISSUE THREAD

**Issue by peterszatmary** [neo-project/neo-devpack-java#6]
I was not able to create hello world smart contract with neo-devpack-java.

1. [maybe documentation is old](http://docs.neo.org/en-us/sc/getting-started-java.html)
2. or in neo-devpack-java are some java files missing

Documentation tells us to extends our smart contracts with **FunctionCode** or **VerificationCode**.

Example from doc

```java
import AntShares.SmartContract.Framework.FunctionCode;
import AntShares.SmartContract.Framework.Services.AntShares.Storage;

public class HelloWorld extends FunctionCode{
    public static byte[] Main(String[] args){
        Storage.Put(Storage.getCurrentContext(), "Greeting to the World", "Hello World!");
        return Storage.Get(Storage.getCurrentContext(),"Greeting to the World");
    }
}
```

Neo-devpack-java doesnt contain this files. 

Link to [Antshares.SmartContract.Framework JAR](http://docs.neo.org/en-us/sc/getting-started-java.html) by documentation is broken.

I found one [issue telling that Antshares.SmartContract.Framework JAR is compiled from neo-devpack-java](https://github.com/neo-project/docs/issues/173).

If so than above mentioned documentation is old or neo-devpack-java is missing java files.

**Comment by erikzhang** [comment]
Yes, the documentation is old.

**Comment by peterszatmary** [comment]
Can i somehow help with the documentation ?

**Comment by erikzhang** [comment]
Yes, of course. You can make pull requests at https://github.com/neo-project/docs

**Comment by zhanglianghui** [comment]
The [Antshares.SmartContract.Framework JAR](http://docs.neo.org/en-us/sc/getting-started-java.html)  has been replaced by org.neo.smartcontract.framework.jar, and will be managed with maven later.
